### PR TITLE
corrects db:migrate and db:seed markdown formatting

### PR DESCRIPTION
**Pull request by abronkema**
Super nitpicky and minor changes to the markdown - there was space missing in `rake:db:migrate` and the `bundle exec rails db:seed` lines.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,22 +80,27 @@ device available in those moments.
...
+
 This app uses Ruby version 2.6.3 and Rails version 5.2.3
 
 ### Setup
-* Clone the repo
-* run `bundle install`
-* run `rails db:create`
-* run `rake:db:migrate`
+
+- Clone the repo
```
Comment (dixpac) [inline README.md:88]: @abronkema can we use [`bin/setup` ](https://github.com/rubyforgood/babywearing/blob/master/bin/setup) ?

Comment (abronkema) [reply README.md:88]: @dixpac - That seems pretty straightforward. Maybe you and I can talk through that in our call!

Comment (DamirSvrtan) [reply README.md:88]: Can we get this one in, with `bin/setup` as @dixpac said and then try to resolve this one as they are conflicting PR's: https://github.com/rubyforgood/babywearing/pull/85

Comment (pollygee) [reply README.md:88]: I'll merge this in as is, then I'll add the `bin/setup` in a separate PR.  Just so we can get this in and start resolving some stuff.

**Pull request merged**